schema/datatypes/hardware_info.fbs

```diff
@@ -49,6 +66,8 @@ table HardwareInfo {
     hardware_address: HardwareAddress;
 
     ip_address: solarxr_protocol.datatypes.Ipv4Address;
+
+    board_type: BoardType;
```

Comment:
If you are going to add a board type, this should be a string identifier not a fixed set. This is because a string allows third parties to describe their board.